src/main/resources/application-default.yml

```diff
@@ -3,8 +3,14 @@ server:
   port: 8075
 
 git:
-  local: tmp/git
+  local: /tmp/git
+  remote-branch: master
 
 app:
   basic-auth-username: user
   basic-auth-password: random-secret
+
+scheduling:
+  push:
+    initial-delay: 10000
+    pause-delay: 30000
```

Comment:
d: I am wondering whether this frequency is too low. I guess we may need to this more often, as otherwise for a service that may only take 10 seconds to be executed we increase the offset by 30 seconds for no obvious reason. I think we can even set this to 5 seconds, as there is a check in there that only does something if there is actually a commit available.

Reply:
Hm i see your point.
That is definitely unnecessary delay, Will play around locally with the value and the jmeter suite to find a good value, would be great if 5s works nicely.